### ISSUE TITLE
Location Entity shows device tracker friendly name instead of state

### ISSUE THREAD

**Issue by Mamouti** [WJDDesigns/Ultra-Vehicle-Card#29]
I can't get the Location Entity field to work, it shows my device tracker entity friendly name instead of its state:
![image](https://github.com/user-attachments/assets/20fca8d6-fbf7-4c3b-b98f-a1737e16795c)

In this case it should display "Chélieu":
![image](https://github.com/user-attachments/assets/2a9a6bbb-164a-4b49-9368-ad5c5d830d1d)

I tried with various location tracker entities (not just the car), it always shows the friendly name.

**Comment by crusell** [comment]
Same for me.

**Comment by ewalk40** [comment]
Same here 2023 Ford Lightning using FordPass

**Comment by WJDDesigns** [comment]
We do not support GOS coordinates yet so right now it should grab the names as places you entered in Home Assistant like Home.

**Comment by Youri3246** [comment]
Same to me.
Car is on location Work and it only show's de friendly name of my car.

**Comment by Mamouti** [comment]
> We do not support GOS coordinates yet so right now it should grab the names as places you entered in Home Assistant like Home.

You probably meant GPS?
I'm sorry but I don't understand, how am I supposed to make it work?

**Comment by WJDDesigns** [comment]
I might add GPS support to show actual location soon.

**Comment by chobbs42** [comment]
There is definitely an issue in this code:

 ```
_renderInfoLine() {
  const locationEntity = this.config.location_entity ? this.hass.states[this.config.location_entity] : null;
  let location = null;


 if (locationEntity) {
    if (locationEntity.state.toLowerCase() === 'not_home') {
      location = 'Not Home';
    } else if (locationEntity.state === 'home') {
      location = 'Home';
    } else {
      // For custom locations, use the friendly name if available, otherwise use the state
      location = locationEntity.attributes.friendly_name || this._capitalizeFirstLetter(locationEntity.state);
    }
  }
```
I suspect the final else should simply be:
```
location = this._capitalizeFirstLetter(locationEntity.state);
```
Or maybe even just:
```
location = locationEntity.state;
```

locationEntity.attributes.friendly_name is returning the friendly name of the device tracker, not the state, if the state isn't 'not_home' or 'home'.

**Comment by WJDDesigns** [comment]
This has been fixed but waiting to push the update since I am fixing a few things.

**Comment by WJDDesigns** [comment]
Fixed in V1.3.1
